File: POOJAVA/src/clasesPOO/Universidad.java
package clasesPOO;

public class Universidad 
{
    String idUni;
    String nombre;
    String direccion;
    String telefono;
    String web;

    public Universidad(String nomb,String we)
    {
        establecerDatos(nomb,we);
    }
    public String ObtenerNombre()
    {
       return nombre;
    }
    public String ObtenerWeb()
    {
       return web;
    }
    void  establecerDatos(String nom,String w)
    {
        nombre=nom;
        web=w;
    }
}

File: POOPHP/app/Model/Universidad.php
<?php
namespace App\Model;
class Universidad
{
    public $idUni="";
    public $nombre="";
    public $direccion="";
    public $telefono="";
    public $web="";
   public function __construct(String $nomb,String $we)
    {
        $this->establecerDatos($nomb,$we);
    }
    public function ObtenerNombre()
    {
        return $this->nombre;
    }
    public function ObtenerWeb()
    {
        return $this->web;
    }
    public function establecerDatos(String $nom,String $w)
    {
        $this->nombre=$nom;
        $this->web=$w;
    }
}
?>